src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/ResourceController.cs

```diff
@@ -149,5 +149,90 @@ public ActionResult LayoutSettings(string org, string app)
             string settings = _appResourceService.GetLayoutSettings();
             return Ok(settings);
         }
+
+        /// <summary>
+        /// Get the layout settings.
```

Comment:
Settings or sets? There is som ambiguity in some of these docs

Reply:
@altinnadmin settings are order inside a specific set. 

Sets are the different sets of layouts. Each containing a settings.

Reply:
The method summary say `Get the layout settings.` but the method is returning a LayoutSet.

Reply:
Fixed